771.py

```diff
@@ -0,0 +1,10 @@
+class Solution:
+    def numJewelsInStones(self, jewels: str, stones: str) -> int:
+        jewels_set = set(list(jewels))
```

Comment:
Redundant step by converting the string to a list and then convert to a set. You can do it in one step as mentioned [here](https://github.com/woke7/leetcode-python/pull/26#discussion_r606714633).